contrib/magnetotelluric_1D/README.md

```diff
@@ -0,0 +1,142 @@
+# Magnetotelluric 1D
+
+<!-- Please write anything you'd like to explain about the forward problem here -->
```

Comment:
Hello Hoel, would you like to document briefly what the problem is about, how to use it (if there's anything special and additional to the "standard" interface), and anything else you think would be beneficial to explain? This document is in Markdown format, so you can write titles starting with `#` (`##` for header 2 and so on) and normal paragraphs with empty lines in between, etc. And please remove all the currently existing content after that.